### ISSUE TITLE
Recommend tldr

### ISSUE THREAD

**Issue by seisman** [seismo-learn/seismology101#110]
I find that [tldr](https://tldr.sh/) is much better than the long man page.

I think we should recommend tldr somewhere.

**Comment by core-man** [comment]
Yes, maybe at the top of comonly-used commands.

**Comment by seisman** [comment]
Where do we put the install instructions? In the OS setup guides?

**Comment by core-man** [comment]
> Where do we put the install instructions? In the OS setup guides?

Yes

**Comment by seisman** [comment]
Easy to install on macos and Fedora, but a little tricky to install on ubuntu.

**Comment by zhaozhiyuan1989** [comment]
> Easy to install on macos and Fedora, but a little tricky to install on ubuntu.

直接 `sudo apt install tldr` 就可以了

**Comment by seisman** [comment]
> > Easy to install on macos and Fedora, but a little tricky to install on ubuntu.
> 
> 直接 `sudo apt install tldr` 就可以了

but it provides `tldr-hs`, not `tldr`, right?

**Comment by zhaozhiyuan1989** [comment]
zhao@ZZY-DELL-G7:~$ tldr
Missing: COMMAND

Usage: tldr [-v|--version] [--update] COMMAND
  tldr Client program
zhao@ZZY-DELL-G7:~$ tldr dos2unix
dos2unix
Change DOS-style line endings to Unix-style.Replaces CRLF with CR.

 - Change the line endings of a file:
   dos2unix {{filename}}

 - Create a copy with Unix-style line endings:
   dos2unix -n {{filename}} {{new_filename}}

**Comment by seisman** [comment]
Weird. The page (https://ubuntu.pkgs.org/20.04/ubuntu-universe-amd64/tldr_0.4.0.1-1build2_amd64.deb.html) says the `tldr` package provides the following files:

![image](https://user-images.githubusercontent.com/3974108/105661487-7fae4000-5e9b-11eb-922a-bdf371a0e87d.png)

@zhaozhiyuan1989 Could you please check if `/usr/bin/tldr-hs` and `/usr/bin/tldr` exits? And if `/usr/bin/tldr` is a symlink to `/usr/bin/tldr-hs`?

**Comment by zhaozhiyuan1989** [comment]
$  ls /usr/bin/tldr
tldr     tldr-hs
 $  ll /usr/bin/tldr
lrwxrwxrwx 1 root root 22 Jan 24 16:50 /usr/bin/tldr -> /etc/alternatives/tldr*
$  file /usr/bin/tldr
/usr/bin/tldr: symbolic link to /etc/alternatives/tldr
$  file /usr/bin/tldr-hs
/usr/bin/tldr-hs: ELF 64-bit LSB executable, x86-64, version 1 (SYSV), dynamically linked, interpreter /lib64/ld-linux-x86-64.so.2, for GNU/Linux 3.2.0, BuildID[sha1]=dba871b56263ac55696370fd636192f84a069678, stripped
